docs/src/components/pricing/WhatToExpect.tsx

```diff
@@ -104,8 +104,10 @@ export default function WhatToExpect() {
             <Typography variant="body2" color="text.secondary">
               Developers contributing changes to the front-end code of a project that include the
               software need an active license. You will need to renew your subscription if you wish
-              to continue active development after the end of your subscription. You can learn more
-              about it in{' '}
+              to continue active development after the end of your subscription.
+              <br />
+              {`You don't need to renew your license if you stopped developing with MUI X Pro or Premium. You
```

Comment:
```suggestion
              {`You don't need to renew your license if you have stopped developing a project that contains MUI X Pro or Premium. You
```

Reply:
This would change the meaning, I think a discussion to continue in https://github.com/mui/mui-store/pull/199.

Reply:
I missed the link to https://github.com/mui/mui-store/pull/199.
```suggestion
              {`You don't need to renew your license if you have stopped active development with MUI X Pro or Premium. You
```